app/src/main/java/com/example/memeapp/presentation/mvp/presenter/MemeAppPresenter.kt

```diff
@@ -30,10 +32,16 @@ class MemeAppPresenter(
         view.showMemeInfo(memeId)
     }
 
-    override fun getMemesFromDataBase() {
+    override fun getMemesFromDatabase() {
+        val realm = Realm.getDefaultInstance()
```

Comment:
this is wrong, you should not access realm from the presentation.

Reply:
Done thx